index.bs

```diff
@@ -1141,15 +1141,18 @@ run the following steps:
 
     ISSUE(httpwg/http-extensions#1593): Note that it's up for discussion whether these character restrictions should also apply to |expires|, |domain|, |path|, and |sameSite| as well.
 
-1. If |name|'s [=string/length=] is 0 and |value| contains U+003D (`=`), then return failure.
-1. If |name|'s [=string/length=] is 0 and |value|'s [=string/length=] is 0, then return failure.
+1. If |name|'s [=string/length=] is 0:
+    1. If |value| contains U+003D (`=`), then return failure.
+    1. If |value|'s [=string/length=] is 0, then return failure.
+    1. If |value|, [=byte-lowercased=], [=byte sequence/starts with=] `__host-` or `__secure-`, then return failure. 
```

Comment:
The proper syntax for this would be (otherwise it's not clear they are Infra byte sequences):
```suggestion
    1. If |value|, [=byte-lowercased=], [=byte sequence/starts with=] \``__host-`\` or \``__secure-`\`, then return failure. 
```
This also applies to the changes below.